pyquil/experiment/_main.py

```diff
@@ -378,7 +372,8 @@ def build_setting_memory_map(self, setting: ExperimentSetting) -> Dict[str, List
         """
         meas_qubits = self.get_meas_qubits()
 
-        in_pt = PauliTerm.from_list([(op, meas_qubits.index(cast(int, q))) for q, op in setting.in_operator])
+        # TODO (andrew): How to use .in_state here instead?
```

Comment:
@notmgsk Any thoughts on this?

Reply:
If we could somehow not need `setting._in_operator()` that would be great, as `setting.in_operator` was deprecated in favor of `setting.in_state`. For now, I just made it protected.